internal/olm/operator/registry/fbcindex/fbc_registry_pod.go

```diff
@@ -46,8 +47,9 @@ const (
 	defaultContainerName     = "registry-grpc"
 	defaultContainerPortName = "grpc"
 
-	defaultConfigMapName = "operator-sdk-run-bundle-config"
-	defaultConfigMapKey  = "extraFBC"
+	defaultConfigMapKey = "extraFBC"
+
+	maxConfigMapSize = 1 * 1024 * 1024
```

Comment:
I rebased the conflicts basing off of master since there were no other PRs that touched this file after https://github.com/operator-framework/operator-sdk/pull/6210 was merged. Can you please point to which PR that you are referring to as separate one? If there is another PR, then we should probably get that in first before doing a cherry-pick of this one.

Reply:
I think this is the other PR: https://github.com/operator-framework/operator-sdk/commit/dd20be405cb09d5cffb7ebbba57042441fadbd63

Reply:
Yep, that's the one

Reply:
https://github.com/operator-framework/operator-sdk/pull/6182

Reply:
@everettraven Fixed this PR after merging https://github.com/operator-framework/operator-sdk/pull/6262, could you take a look again?